src/pages/chainlink-nodes/configuring-nodes.mdx

```diff
@@ -71,7 +71,7 @@ You can generate the `config.toml` file using the `chainlink config dump` comman
 
 After you have the `config.toml` file and the `secrets.toml` files on the host system, you can validate these files using a temporary Docker container.
 
-1. Validate the configuration:
+1. Validate the configuration by running the `config validate` command in the Docker container. This command changes to `node validate` when you upgrade your node to version `2.0.0` or later.
```

Comment:
Could you use the code tabs here to show both commands, showing 2.0.0+ as the default on the left side?

Reply:
This section is specific to updating the config to TOML, which can only be done on v1.13.x. The other docs are updated to use the new syntax. The migration doc is the one exception where we need to mention the old commands.